### DISCUSSION TITLE
One Corner not good with ABL 5x5 and 3x3

### DISCUSSION THREAD

**Discussion by DJCAlex** [Jyers/Marlin#1165] [Q&A] [answered]
Hello,

i have an issue with the ABL. I tried the 5x5 and 3x3 BL Touch Versions. So the issue is that when i do a calibration prints with 5 single layer squares ( one in each corner and one in the center) the front left one always looks like the bed is too far away from the nozzle (lines not conected) The other 4 squares look fine. The printer seems to build the mesh fine and I can see the z axis have small adjustmens during the print, so the mesh seems to be loading. I have M420 S1 in my start code after G28. 

If i manually raise said front left corner of the bed slightly and build i new mesh i can also see the mesh chaning, i.e. the front left coming up and the back right going down, so it seems the probe also works correctly. Print result is still the same though, front left looks like bed is to far away, rest is fine.

The issue is bigger with the 5x5 grid, with the 3x3 grid there is still a clearly visual difference between the front left and all the other corners + center

I also tried UBL. UBL seems to work significantly better, so what could be the cause of that mentioned issue with the bilinear 5x5 and 3x3 Grid? 

The Mesh itself seems to be reasonable... All the corners are a bit lower than the center, but the corners are around the same height, around 0.05 within each other was the best i got so far

**Comment by DJCAlex** [comment]
Quick questions, since I'am still trouble shooting:

When i want to create a new mesh before every print with the 5x5 Bilinear version... What is the correct startcode? Google did not give me any answer specifically to the jyers firmware... do i only need G29 after homing? Or anything else?

**Reply by Jyers** [reply]
Either G29 or M420 S1 will do

**Comment by Akujiproxy** [comment]
I have this exact same issue, I level the bed manually and then create a new mesh, the mesh always goes to around -12 at the top right and when I print a first layer test the top right is squished but the bottom right is too far away. It literally seems like the ABL messed things up.

**Comment by Jyers** [comment, marked as answer]
Hopefully you got this working, if not maybe try the latest source and see if that helps :)
